net-vpn/i2p/i2p-0.9.33.ebuild

```diff
@@ -0,0 +1,144 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="6"
+
+inherit eutils java-pkg-2 java-ant-2 systemd user
```

Comment:
Hmm, are you sure you're using eutils here? I don't see any call but I might be missing it.

Reply:
Hum, you're right; `repoman` didn't catch it, that's weird. Done.